<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditServeurWidget</class>
 <widget class="QWidget" name="AuditServeurWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuditServeurWidget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>34</y>
     <width>721</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Num√©ro de port </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBoxPort">
      <property name="minimum">
       <number>1000</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>1025</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="pushButtonLancement">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Lancement Serveur</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Choissisez</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>personnage</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Scores</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Parties</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Informations</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTextEdit" name="textEditInfo">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButtonQuitter">
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>AuditServeurWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>786</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
